pkg/base1/cockpit.js

```diff
@@ -1853,6 +1854,29 @@ function full_scope(cockpit, $, po) {
             return new DBusProxies(self, cache, String(iface), String(path_namespace));
         };
 
+        self.poll = function poll(path, iface) {
```

Comment:
I think this should be implemented as a call through to cockpit_dbus_cache_poke() instead. We could have a dbus JSON "poke" message. The actual property changes would then flow back up through the usual "notify" channels.